buildconfigs/kustomization.yaml

```diff
@@ -0,0 +1,228 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: okd
+
+patches:
+  - patch: |-
+      - op: add
+        path: "/spec/source/images/-"
+        value:  
+           from:
+              kind: "ImageStreamTag"
+              name: "release:builder"
+           as:
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.12'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.18-openshift-4.11'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.10'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.16-openshift-4.9'
+              - 'registry.ci.openshift.org/ocp/builder:rhel-8-etcd-golang-1.16'
+              - 'registry.ci.openshift.org/openshift/release:golang-1.17'
+    target:
+      kind: BuildConfig
+      labelSelector: "builder-replacement!=skip"
+  - patch: |-
+      - op: add
+        path: "/spec/resources"
+        value:
+          limits:
+            memory: 2Gi
+            cpu: 2000m
+          requests:
+            memory: 1Gi
+            cpu: 1000m
```

Comment:
This is brilliant, I think we want some buildconfigs labelled as "large" too (hyperkube rebuild may use even more)

Reply:
Yes, I have some changes locally already to have a few classes of resource requirements to address what you say